keyboards/rotr/kb.c

```diff
@@ -0,0 +1 @@
+#include "kb.h"
```

Comment:
This file should be renamed `rotr.c`, also...
```suggestion
#include "rotr.h"
```